### DISCUSSION TITLE
Provided my dated college ID card still not getting verified.

### DISCUSSION THREAD

**Discussion by AMISH-KHAN** [community/community#55053] [GitHub Education]
### Hi everyone,

I am a student currently pursuing a three year degree course and I have submitted my College Id card in which the date is mentioned that is 2022-25 , but it is getting rejected with a message "Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 10, 2023 ". Please help me with this is issue.

**Comment by Ntshangase** [comment]
Hello @AMISH-KHAN 

Is your university email address already verified on your GitHub profile? After it is verified, re-apply.

**Comment by samus-aran** [comment]
Hi @AMISH-KHAN 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍
